package.json

```diff
@@ -30,31 +29,12 @@
   },
   "type": "commonjs",
   "main": "dist/index.js",
-  "module": "dist/esm/index.mjs",
-  "types": "dist/index.d.ts",
+  "module": "dist/index.mjs",
   "exports": {
-    "./dist/*": {
-      "types": "./dist/*.d.ts",
-      "require": "./dist/*.js",
-      "import": "./dist/esm/*.mjs",
-      "default": "./dist/*.js"
-    },
-    "./bin/*": {
-      "require": "./bin/*.js",
-      "import": "./bin/*.mjs",
-      "default": "./bin/*.js"
-    },
     ".": {
-      "types": "./dist/index.d.ts",
       "require": "./dist/index.js",
-      "import": "./dist/esm/index.mjs",
-      "default": "./dist/index.js"
-    },
-    "./*": {
-      "types": "./dist/*.d.ts",
-      "require": "./dist/*.js"
-    },
-    "./package.json": "./package.json"
```

Comment:
**issue (blocking):** the package.json is explicitly exposed, so someone can use e.g.
```ts
import { version } from 'node-pg-migrate/package.json'
```